pyomo/core/base/PyomoModel.py

```diff
@@ -217,11 +218,27 @@ def load_from(self,
         # If there is a warning, then print a warning message.
         #
         if (results.solver.status == SolverStatus.warning):
+            if hasattr(results.solver, "termination_condition") and (
+                results.solver.termination_condition
+                == TerminationCondition.infeasible
+            ):
```

Comment:
Is there any reason you are only including the termination condition if it is infeasible?  Also, why remove the message formatting?

Would you consider the following solution:
```python
            tc = getattr(results.solver, 'termination_condition', None)
            msg = getattr(results.solver, 'msg', None)
            logger.warning(
                'Loading a SolverResults object with a '
                'warning status into model=%s;\n'
                '    termination condition=%s\n'
                '    message from solver=%s'
                % (instance.name, tc, msg))
```

Reply:
Whoops missed this. Looks like you addressed this in #1752 so thanks!

Good point—no good reason to only include termination condition if infeasible. 

And I'm not quite sure I understood the point of the way it was formatted before and marginally preferred the update I made but don't feel strongly at all about that one (I assume just convention pyomo has adopted?)